src/actions/publish.js

```diff
@@ -95,6 +95,11 @@ async function storeJustPublishedProfileData(
   sanitizedInformation,
   prepublishedState: State
 ): Promise<void> {
+  if (process.env.NODE_ENV === 'test' && !window.indexedDB) {
+    // bailout early in tests
```

Comment:
Suggestion: It would be clearer to change this comment from a "what" comment to a "why" comment.